packages/imagemagick6.rb

```diff
@@ -63,7 +58,7 @@ def self.build
            --with-perl \
            --with-rsvg \
            --with-x"
-    system 'make'
+    system "make -j#{CREW_NPROC}"
```

Comment:
This is not necessary.  The default value of `-j` is set to this value.  See `lib/package.rb`.

Reply:
Fine ! I'll remove it from imagemagick too then.